cmd/ls/lshelp/lshelp.go

```diff
@@ -8,8 +8,8 @@ There are several related list commands
 
   * ` + "`ls`" + ` to list size and path of objects only
   * ` + "`lsl`" + ` to list modification time, size and path of objects only
-  * ` + "`lsd`" + ` to list directories only
-  * ` + "`lsf`" + ` to list objects and directories in easy to parse format
+  * ` + "`lsd`" + ` to list directories only, sorted ascending
+  * ` + "`lsf`" + ` to list objects and directories in easy to parse format, sorted ascending
```

Comment:
And likewise this one won't be sorted unless `-R` is passed

Reply:
In my tests with 1.52, neither command recursed, so I haven't used `-R` or `--max-depth` on either. Docs say "The other list commands lsd,lsf,lsjson do not recurse by default - use "-R" to make them recurse.". I ran `lsd` and `lsf` against the root dir, where I had the files I wanted sorted, and directories to recurse into as well (but they weren't recursed into). Beyond that, I don't know :)